database/migrations/2019_09_05_153524_create_notifications_attribs_index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateNotificationsAttribsIndex extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('notifications_attribs', function (Blueprint $table) {
+            $table->index(['notifications_id','user_id'], 'notifications_id_user_id');
```

Comment:
What's the reason for naming the index manually?

Reply:
Had assumed it was necessary for the dropIndex, removing.